src/app/dropdown/dropdown-angular-positioning.demo.html

```diff
@@ -5,37 +5,106 @@
-            <a href="javascript://" clrDropdownItem>Action 2</a>
-            <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
-            <div class="dropdown-divider"></div>
-            <a href="javascript://" clrDropdownItem>Link 1</a>
-            <a href="javascript://" clrDropdownItem>Link 2</a>
-        </div>
-    </clr-dropdown>
-    <clr-dropdown [clrMenuPosition]="'bottom-left'">
-        <button class="btn btn-outline-primary" clrDropdownToggle id="current">
-            Dropdown
-            <clr-icon shape="caret down"></clr-icon>
-        </button>
-        <clr-dropdown-menu *clrIfOpen>
-            <label class="dropdown-header">Dropdown header</label>
-            <a href="javascript://" clrDropdownItem>Action 1</a>
-            <a href="javascript://" clrDropdownItem>Action 2</a>
-            <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
-            <div class="dropdown-divider"></div>
-            <a href="javascript://" clrDropdownItem>Link 1</a>
-            <a href="javascript://" clrDropdownItem>Link 2</a>
-        </clr-dropdown-menu>
-    </clr-dropdown>
+<div class="clr-example">
+    <p>
+        Static menu, no <code class="clr-code">*clrIfOpen</code> (deprecated):
+        <clr-dropdown>
+            <button class="btn btn-outline-primary" clrDropdownToggle>
+                Dropdown
+                <clr-icon shape="caret down"></clr-icon>
+            </button>
+            <div class="dropdown-menu">
+                <label class="dropdown-header">Dropdown header</label>
+                <a href="javascript://" clrDropdownItem>Action 1</a>
+                <a href="javascript://" clrDropdownItem>Action 2</a>
+                <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
+                <div class="dropdown-divider"></div>
+                <a href="javascript://" clrDropdownItem>Link 1</a>
+                <a href="javascript://" clrDropdownItem>Link 2</a>
+            </div>
+        </clr-dropdown>
+        <clr-dropdown class="top-right">
+            <button class="btn btn-outline-primary" clrDropdownToggle>
+                Dropdown
+                <clr-icon shape="caret down"></clr-icon>
+            </button>
+            <div class="dropdown-menu">
+                <label class="dropdown-header">Dropdown header</label>
+                <a href="javascript://" clrDropdownItem>Action 1</a>
+                <a href="javascript://" clrDropdownItem>Action 2</a>
+                <a href="javascript://" class="disabled" clrDropdownItem>Disabled Action</a>
+                <div class="dropdown-divider"></div>
+                <a href="javascript://" clrDropdownItem>Link 1</a>
+                <a href="javascript://" clrDropdownItem>Link 2</a>
+            </div>
+        </clr-dropdown>
+    </p>
+    <p>
+        Menu component, no <code class="clr-code">*clrIfOpen</code> (not recommended):
```

Comment:
So if you dont use `clrIfOpen` the menu would remain in the DOM and just be hidden using CSS right?